rudder-web/src/test/resources/api/api_v2.yml

```diff
@@ -0,0 +1,15 @@
+---
+description: Simple test of api/status - just "ok"
+method: GET
+url: /api/status
+response: 
+  type: text #optionnal, by default json
+  code: 200
+  content: 'OK'
+---
+method: GET
+url: /api/techniqueLibrary/reload
+response:
+  type: plop
```

Comment:
Ok, it's better to have failing tests, but now I can't commit them because they will fails the test... not sur how to handle that.